pages/a/decred-101.vue

```diff
@@ -45,23 +45,52 @@
 
               <h2>Decred is Sound Money - This is El Captain edit</h2>
               <p>
-                In current times, traditional paper money or "fiat money"
-                (dollars, euros, etc.) is controlled by banks, which create new
-                money as credit. The money supply is controlled to target
-                specific economic conditions, usually including a degree of
-                inflation in prices. Essentially, anyone who is not benefitting
-                directly from the creation of the new money is losing out.
+                To understand "sound" (not able to be manipulated) money, it is first 
+                important to understand the history of money and money as a funtion of its 
+                characteristics. For the purpose of brevity we will not detail money's history
+                and focus more on the characteristics of money and what makes Decred Sound Money. 
+                For a more in depth overview of the history of money, we recommend 
+               <a
+                    href="https://www.investopedia.com/articles/07/roots_of_money.asp"
+                    target="_blank"
+                    class="underline"
+                    >Investopedia's "The History of Money"</a
```

Comment:
It's good to link to other sources, but I'm also a bit worried if it won't take people's attention away from what we want -> read all the articles and buy Decred. I'm not going to be too opinionated about leaving or throwing out links here, we will measure it and see how people behave.